### ISSUE TITLE
Probably no need for OCR software

### ISSUE THREAD

**Issue by sonofmun** [ThomasK81/vagrantboxes#2]
@ThomasK81 Since it doesn't look like we will be doing anything with OCR during the week, I think you can cross the OCR software off of the list of things to install in the box.

**Comment by ThomasK81** [comment]
That is not true. There was a simple misunderstanding. Uwe's talk was not dropped, it only dropped out because people were working with 2 Google docs at the same time!
